src/widgets/sort-by/__tests__/sort-by-test.js

```diff
@@ -88,12 +88,6 @@ describe('sortBy()', () => {
     expect(ReactDOM.render.mock.calls[0][0]).toMatchSnapshot();
   });
 
-  it('sets the underlying index', () => {
```

Comment:
This was an implementation detail that was already tested at the connector level.